Solution Items/.gitignore

```diff
@@ -0,0 +1,2 @@
+﻿# Ignore compressed dictionaries
```

Comment:
It's in the `.gitignore`, so it's clear that this line ignores something. Simply describe the kind of file that is being ignored.